src/resource-graph/azext_resourcegraph/_params.py

```diff
@@ -30,19 +28,16 @@ def load_arguments(self, _):
     with self.argument_context('graph query') as c:
         c.argument('graph_query', options_list=['--graph-query', '--q', '-q'], required=True,
                    completer=get_generic_completion_list(_QUERY_EXAMPLES), help='Resource Graph query to execute.')
-        c.argument('first', options_list=['--first'], required=False, type=int, default=100,
-                   help='The maximum number of objects to return. Accepted range: 1-5000.')
-        c.argument('skip', options_list=['--skip'], required=False, type=int, default=0,
-                   help='Ignores the first N objects and then gets the remaining objects.')
+        c.argument('first', options_list=['--first'], required=False, type=int, default=None,
+                   help='The maximum number of objects to return. Accepted range: 1-1000. Default value is 100.')
+        c.argument('skip', options_list=['--skip'], required=False, type=int, default=None,
+                   help='Ignores the first N objects and then gets the remaining objects. Default value is 0.')
```

Comment:
Same question as above

Reply:
Same as above - we need to know whether user overwrote this value or left it blank. The default value is set in customs.py